ui/src/components/Issuer/Issuer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Switch, Route } from 'react-router-dom'
+
+import Page from '../common/Page'
+import WelcomeHeader from '../common/WelcomeHeader'
+
+import IssuerSideNav from './IssuerSideNav'
+import IssueAsset from './IssueAsset'
+import IssuedToken from './IssuedToken'
+
+type Props = {
+    onLogout: () => void;
+}
+
+const Issuer: React.FC<Props> = ({ onLogout }) => (
+    <Switch>
+        <Route path='/role/issuer/issue-asset'>
```

Comment:
I also used absolute paths at first, but found what I think is a good pattern for making them relative. Here in the issuer component, you can do 

```
const { path, url } = useRouteMatch();
```

Then, build paths off the path value like ``<Route path={`${path}/issue-asset`}>``

For the sidenav links, pass the `url` into the sidenav and then you can do the same thing - ``<Menu.Item to={`${url}/issue-asset`} ... >`` for example

And you can use just `${url}` with nothing added, which will appropriately resolve to `/role/issuer`

Reply:
ur right I will make that change - much better